packages/ethereum/sdk/src/config/utils.ts

```diff
@@ -0,0 +1,73 @@
+import { ZERO_ADDRESS } from "@rarible/types"
+import { merge } from "lodash"
```

Comment:
lodash package increases the bundle. I suggest using a simple implementation of deep merging function.